Dockerfile.solrwayback

```diff
@@ -0,0 +1,40 @@
+# This dockerfile configures a vanilla tomcat container
+# with solrwayback installed and configured with properties
+# from solrwayback bundle.
+#
+# See https://hub.docker.com/_/tomcat for details on how
+# to configure tomcat.
+
+ARG SOLRWAYBACK_VERSION=4.4.2
+ARG SOLRWAYBACK_TOMCAT_VERSION=8.5.60
+ARG TOMCAT_TAG=8.5-jdk8-temurin-jammy
+
+FROM ubuntu:22.04 as solrwayback-bundle
+
+ARG SOLRWAYBACK_VERSION
+
+RUN apt-get update \
+    && apt-get install --quiet --assume-yes wget unzip python3
+
+WORKDIR /build
+COPY fetch_solrwayback_bundle.py .
+
+RUN python3 fetch_solrwayback_bundle.py \
+    --solrwayback-version ${SOLRWAYBACK_VERSION} \
+    --destination /app
+
```

Comment:
Create new `RUN` step that unzip separately from the download of the bundle to preserve the cached bundle download layer.

Reply:
Solved by https://github.com/nlnwa/solrwayback-adaption/pull/17/commits/f56b80e59e560d1e3e6b29a36b37cd76163c205d